migrations/20210508000000_form_db.js

```diff
@@ -0,0 +1,11 @@
+exports.up = knex =>
+  knex.schema.createTable('report_log', table => {
+    table.increments('id').primary('pk_audit')
+    table.timestamp('timestamp').notNullable().defaultTo(knex.fn.now())
+    table.string('username', 50).notNullable()
+    table.bigInteger('report_id').notNullable()
+    table.string('action', 50).notNullable()
+    table.jsonb('details').nullable()
+  })
```

Comment:
based on the licences version

Reply:
Think you might need some indexes on this table.  report_id? username? timestamp?